ipa-core/src/net/error.rs

```diff
@@ -75,7 +77,7 @@ impl Error {
         let status = resp.status();
         assert!(status.is_client_error() || status.is_server_error()); // must be failure
         let (endpoint, body) = resp.into_parts();
-        hyper::body::to_bytes(body)
+        axum::body::to_bytes(body, usize::MAX)
```

Comment:
`usize::MAX` is concerning here. Is there a way to put a limit here?

Reply:
What number do you think?

Reply:
we chatted offline, a few dozen MBs seems reasonable to me